packages/plugins/AddressSearch/src/components/Results.vue

```diff
@@ -26,14 +28,37 @@
       </v-subheader>
       <v-list-item
         v-for="(feature, innerDex) in features"
+        :id="
+          ['polar-plugin-address-search-results-feature', index, innerDex].join(
+            '-'
+          )
+        "
         :key="['results-feature', index, innerDex].join('-')"
+        tag="li"
+        tabindex="-1"
         :class="{
           'polar-plugin-address-search-hidden-result':
             innerDex >=
             (openCategories.includes(category)
               ? Number.MAX_SAFE_INTEGER
               : limitResults),
         }"
+        @keydown.down.prevent.stop="
```

Comment:
This stops working when NVDA is turned on.

Reply:
It still works, the visual focus is lost though.

While looking into this problem, I didn't manage to find a solution where the visible arrow-key-focus exists while using NVDA.  
Do you have a suggestion in which direction a solution may lie or do you think that this is negligible as a ScreenReader user most likely will focus (HA) on what the ScreenReader is reading?

Reply:
I think both visual and speech information are required. However, I now consider this resolved:

After some research I figured out it's a skill issue on our end. ScreenReader users seem to be used to change the arrow key mode by pressing "Insert+Space". I tested that and it then works as intended.

🎩